lib/debugger-wrapper.js

```diff
@@ -97,11 +97,15 @@ function resumeSnoozedBreakpoints() {
     const bpData = breakpointsMap[breakpointId];
     setBreakpointOnFunction(bpData.moduleUrl, bpData.moduleInfo, bpData.methodName, bpData.methodLocation);
   });
-  suspendedBreakpointIds = [];
+  suspendedBreakpointIds.length = 0;
```

Comment:
Does this leak?

Reply:
it shouldn't - 
```
> const arr = [1,2,3,4]
undefined
> arr
[ 1, 2, 3, 4 ]
> arr.length
4
> arr.length = 0
0
> arr
[]
```
I don't know how to check better, but it looks like everything inside the array is gone and unreferenced